backend/cfg/cfg.ml

```diff
@@ -195,6 +195,16 @@ let intop (op : Mach.integer_operation) =
   | Icomp cmp -> intcomp cmp
   | Icheckbound -> assert false
 
+let floatop op =
+  match (op : Mach.float_operation) with
+  | Inegf -> " negf "
+  | Iabsf -> " absf "
+  | Iaddf -> " addf "
+  | Isubf -> " subf "
+  | Imulf -> " mulf "
+  | Idivf -> " divf "
+  | Icompf cmp -> Printf.sprintf "compf %sf" (Printcmm.float_comparison cmp)
```

Comment:
It is for consistency with Printcmm, which appends "f" to float comparison operators distinguish them from int comparison operators in the output (which is probably redundant because the type of operands should usually be printed).